### DISCUSSION TITLE
Issues activating this with txs in nextjs project.

### DISCUSSION THREAD

**Discussion by Saga4** [moalamri/vscode-inline-fold#36] [Show and tell]
I was looking for some extension to have a cleaner look while using tailwinds. 
Now how do I enable this extension. In regex I have already added ```className="([a-zA-Z0-9-\s]+)(?=")```. While trying to save and open any file it isn't working. 
Please let me know if I am missing anything here.

**Comment by moalamri** [comment]
If you are using reactjs the default regex should work, what is  the language id of nextjs in VS Code?

**Reply by Saga4** [reply]
Thanks for responding. We are using typescript in our project.

**Reply by moalamri** [reply]
> Thanks for responding. We are using typescript in our project. 

Have you tried the default regex? You can go to settings UI and press on the settings icon next to the inline fold regex and select reset. It should work if the language id is included in the supported languages list (which is supported by default if the language is typescript)

**Reply by Saga4** [reply]
T

> 

Reloaded, reset the settings and again reloaded the VS code and things didn't work.  Tried restarting the application as well. 
Let me know how can I debug this further.

**Reply by moalamri** [reply]
Yes please, what is `txs`?. You say it is for nextjs. 
Is it only for nextjs? Because I need to figure the language ID so you can add to settings
